common/openapi.yml

```diff
@@ -576,8 +576,10 @@ components:
 
     PipelineRunOutput:
       description: |
-        The output of a successful pipeline run.
-      type: string
+        The output(s) of a successful pipeline run.
+      type: object
```

Comment:
do we want to have a more fleshed out definiton for this?  I think that fleshed out thing will still have an object like this for the outputs but im curious if there is anything we want all result outputs to share that wouldnt already be in this payload.  Like pipeline name or version or anythign like that.

Reply:
good call - TSPS-259 created